test/core/fixtures/transformation/es7.function-bind/call/expected.js

```diff
@@ -0,0 +1,8 @@
+"use strict";
+
+var _context;
+
+ns.obj.func.call(ctx);
+(_context = ns.obj).func.call(_context);
+
+ns.obj1.func.call(ns.obj2);
```

Comment:
It's still very important to get correct semantics. No point going half way. You'll also have to memoise single identifier references that aren't bound anywhere in the file (`generateMemoisedReference` will be of use here) since referencing it may trigger a globally set getter.